src/backend/apps/utils/middleware.py

```diff
@@ -0,0 +1,23 @@
+from django.middleware.common import CommonMiddleware
+from django.conf import settings
+from django.urls import is_valid_path
+
+
+class CustomCommonMiddleware(CommonMiddleware):
```

Comment:
Rename "nuxt index redirect" or something?

Reply:
Came up for a better name for it!